src/ctap/hid/mod.rs

```diff
@@ -503,68 +504,25 @@ impl CtapHid {
             payload: response,
         }
     }
+
+    #[cfg(test)]
+    pub fn new_initialized() -> (CtapHid, ChannelID) {
```

Comment:
Ah right, this doesn't make sense from this PR's perspective. It will be used from the outside, when HID gets instantiated twice. I'll then use it for testing structs that instantiate HID.

Reply:
Ah ok. SGTM then.